charts/csi-powermax/Chart.yaml

```diff
@@ -1,26 +1,25 @@
 apiVersion: v2
-appVersion: "2.10.1"
+appVersion: "2.11.0"
 name: csi-powermax
-version: 2.10.1
+version: 2.11.0
 description: |
   PowerMax CSI (Container Storage Interface) driver Kubernetes
   integration. This chart includes everything required to provision via CSI as
   well as a PowerMax StorageClass.
 type: application
 kubeVersion: ">= 1.23.0"
 # If you are using a complex K8s version like "v1.23.3-mirantis-1", use this kubeVersion check instead
-# WARNING: this version of the check will allow the use of alpha and beta versions, which is NOT SUPPORTED
-# kubeVersion: ">= 1.23.0-0"
+# kubeVersion: ">= 1.23.0-0
```

Comment:
Why are we leaving these comments in? And if we're going to leave the last one as an option, shouldn't we close the quotation marks around it instead of leaving it open?

Reply:
It looks like all driver charts don't close the quotation mark but I'm not sure why that is. We should determine if this is ok or if it needs to be addressed.

Reply:
@ChristianAtDell The comments are just for illustration to help customers get past the safety checks. Some CO platforms like Mirantis report version with a hyphen. Unless an update is made here to reflect the right format, these checks will not pass. Hence the sample was added. I am not sure on the quotes/no quotes, but I think quotes are probably better option. Hope this helps.

Reply:
Fixed in #473 473